### PR TITLE
Functional tests - Refacto 'BO > Customer service'

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 8.1.x
| Description?      | Refacto 'BO > Customer service'
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | Nightly is :green_circle: => https://github.com/nesrineabdmouleh/testing_pr/actions/runs/4562368132
| Fixed ticket?     | no
| Related PRs       | no
| Sponsor company   |

### CODE REVIEW

**Review by boubkerbribri — CHANGES_REQUESTED** [review]
There is some conflicts to resolve

File: tests/UI/data/faker/message.ts [hunk 1]
```diff
@@ -37,6 +39,9 @@ export default class MessageData {
     /** @type {string} Firstname of the customer */
     this.lastName = messageToCreate.lastName || faker.name.lastName();
 
+    /** @type {string} employee to forward the message */
+    this.employee = messageToCreate.employee || `${this.firstName.slice(0.1)}. ${this.lastName}`;
```
Comment (Progi1984) [inline tests/UI/data/faker/message.ts]:
```suggestion
    /** @type {string} employee to forward the message */
    this.employeeName = messageToCreate.employee || `${this.firstName.slice(0.1)}. ${this.lastName}`;
```

I think you can rename the property or change the property with a EmployeeData type.

**Review by Progi1984 — COMMENTED** [review]
A small feedback about `message.d.ts`

File: tests/UI/data/faker/message.ts (hunk 1 above)
Comment (nesrineabdmouleh) [reply tests/UI/data/faker/message.ts]: Done :+1:

**Review by Progi1984 — APPROVED** [review]

**Review by boubkerbribri — APPROVED** [review]

**Review by nicosomb — APPROVED** [review]

**Pull request merged**